manifests/d/DMM/GamePlayer/5.0.119/DMM.GamePlayer.installer.yaml

```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: DMM.GamePlayer

```

Comment:
```suggestion
PackageIdentifier: DMM.GamePlayer.5
```
This Package Installs Side-By-Side and should be under a different PackageIdentifier than the current one to avoid conflicts
![image](https://user-images.githubusercontent.com/12611259/160197600-d3ac3103-c2d5-4881-8483-7b05f4398226.png)

Reply:
it is a compulsory update, v.2.5.8 no longer works. So I removed the old version. from the repo.